media/Jsontracer/processData.js

```diff
@@ -50,35 +50,7 @@ import renderDashboard from './dashboard.js';
 
 const setData = document.querySelector('.set-data');
 
-export default function openFileSelector() {
-  const input = document.createElement('input');
-  input.type = 'file';
-  input.accept = 'text/plain';
-  input.onchange = (event) => {
-    setFileName(event.target.files[0].name);
-    processFile(event.target.files[0]);
-  };
-  input.click();
-}
-
-function setFileName(name) {
-  const fileName = document.querySelector('.file-name');
-  fileName.innerText = name;
-}
-
-function processFile(file) {
-  const reader = new FileReader();
-  reader.onload = () => {
-    const data = JSON.parse(reader.result);
-    processData(data.traceEvents);
-
-    // set data to DOM
-    setData.dataset['displayTimeUnit'] = data.displayTimeUnit;
```

Comment:
Since this part has been deprecated, the features which are using `displayTimeUnit` makes bugs :sob:

For example, when user hit a block in jsontracer, `ts` and `dur` item are shown as `NaN undefined`. That's caused by following lines.
https://github.com/Samsung/ONE-vscode/blob/9324f977bae56cea9ea3e5f6404c89d43bc8b0d1/media/Jsontracer/detail.js#L76-L80
`const displayTimeUnit` became undefined, since it has not been set yet. T^T 

I'll post a new issue tomorrow :)